pkg/lifecycle/startup/startup_windows.go

```diff
@@ -89,7 +90,17 @@ func ensureNetworkForOS(ctx context.Context, c client.Interface, nodeName string
 				return err
 			}
 			_, err = windows.SetupVxlanNetwork(networkName, subnet, uint64(vni), logrus.WithField("subnet", subnet.String()))
```

Comment:
I suggest we add a `if err !=nil` here, so that we can catch error earlier if network is not created properly.